### PR TITLE
Remove the ScriptCompletion level type

### PR DESCRIPTION
**Pull request by joshlory**
This removes the `ScriptCompletion` level type added in https://github.com/code-dot-org/code-dot-org/pull/8203.  These will instead be implemented as horizontal puzzle dots directly in the script.

### CODE REVIEW

File: dashboard/app/models/game.rb [hunk 1]
```diff
@@ -224,7 +220,7 @@ def self.setup
         Gamelab:gamelab
         LevelGroup:level_group
         FreeResponse:free_response
-        ScriptCompletion:script_completion
+        NotUsed:not_used
```
Comment (joshlory) [inline dashboard/app/models/game.rb:223]: Can't remove this entry because it would shift the IDs of the `Game` instances that occur after.

**Comment by mehalshah** [comment]
If there's no longer a level for a script completion, how will the module assignment know when the user has completed all the levels they are supposed to complete? Will they be associated directly with the stage?

**Comment by mehalshah** [comment]
factories.rb has a script_completion_task factory - that should be removed and I imagine a lot of tests that use it need to be updated.

**Comment by joshlory** [comment]
Whoops, yeah, every test ever failed 😒 .  I'll fix!

**Comment by joshlory** [comment]
@mehalshah ready for another pass.

**Comment by mehalshah** [comment]
Tests are still failing though I'm suspicious as to whether they have to do with your change. Might want to restart them.

**Comment by joshlory** [comment]
Should be fixed now.

**Comment by mehalshah** [comment]
LGTM

**Comment by Hamms** [comment]
@joshlory can the ScriptCompletion class be safely removed now? https://github.com/code-dot-org/code-dot-org/blob/950d2ba80faf46d21027eb0c1194eebf5c91df94/dashboard/app/models/levels/script_completion.rb

**Pull request merged**